distribution/feature-packs/server-feature-pack/src/main/resources/licenses/rh-sso/licenses.xml

```diff
@@ -903,6 +903,18 @@
         </license>
       </licenses>
     </other>
+    <other>
+      <description>Moment.js</description>
+      <locations>
+        <file>themes/keycloak-preview/account/resources/node_modules/moment/min/moment-with-locales.min.js</file>
+      </locations>
+      <licenses>
+        <license>
+          <name>MIT License</name>
+          <url>https://raw.githubusercontent.com/moment/moment/develop/LICENSE</url>
+        </license>
+      </licenses>
+    </other>
```

Comment:
I think we need to do the same for Keycloak as well (the other licenses.xml).

Reply:
It should be there:

```
aszczucz@tyrfing ~/s/k/keycloak3 (KEYCLOAK-12904)> git grep Moment.js
distribution/feature-packs/server-feature-pack/src/main/resources/licenses/keycloak/licenses.xml:      <description>Moment.js</description>
distribution/feature-packs/server-feature-pack/src/main/resources/licenses/rh-sso/licenses.xml:      <description>Moment.js</description>
```